hera_sim/sigchain.py

```diff
@@ -1086,12 +1086,15 @@ def build_coupling_matrix(
             else:
                 power_beam = uvbeam.copy()
                 power_beam.efield_to_power()
-                if power_beam.interpolation_function is None:
-                    power_beam.interpolation_function = pixel_interp
+                # if power_beam.interpolation_function is None:
+                #    power_beam.interpolation_function = pixel_interp
```

Comment:
It looks like this is now taken care of in the call to `interp`. Shouldn't these lines be removed instead of commented out?

Reply:
yes.